jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/comments/CaseCommentsViewImpl.html

```diff
@@ -1,30 +1,62 @@
 <!DOCTYPE html>
-<div data-field="comments" class="card-pf">
+<div class="card-pf" data-field="comments">
     <div class="card-pf-heading">
+        <div class="card-pf-time-frame-filter" data-field="search-actions">
+            <button class="btn btn-link kie-btn-link" type="button" data-field="sort-alpha-asc">
```

Comment:
Would be nice to add a title for this to indicate the action, something like: "Sort by date asc"